nmdc_runtime/api/endpoints/queries.py

```diff
@@ -127,6 +133,28 @@ def _run_query(query, mdb) -> CommandResponse:
                     status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
                     detail="Failed to back up to-be-deleted documents. operation aborted.",
                 )
+    elif q_type is UpdateCommand:
+        collection_name = query.cmd.update
+        if collection_name not in nmdc_schema_collection_names(mdb):
+            raise HTTPException(
+                status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+                detail="Can only update documents in nmdc-schema collections.",
+            )
+        update_specs = [
+            {"filter": us.q, "limit": 0 if us.multi else 1} for us in query.cmd.updates
```

Comment:
I don't understand what the `.multi` property does yet. This is after having looked at the `class UpdateStatement(BaseModel):` class definition. I'll keep reading—maybe it will become more clear to me.

Reply:
if `update_specs.multi` is `True` the update will apply to all query results for the update query. (eg `limit: 0`)  

<img width="821" alt="image" src="https://github.com/microbiomedata/nmdc-runtime/assets/26777878/71b84b4a-8a45-47c1-ab55-f37754f6f572">

docs: https://www.mongodb.com/docs/v6.0/reference/command/update/#std-label-update-command-output
see also: https://www.mongodb.com/docs/v6.0/reference/command/update/#std-label-multi-update-failures